sorc/ncep_post.fd/SURFCE.f

```diff
@@ -940,7 +941,7 @@ SUBROUTINE SURFCE
       ENDIF
 !
 !     ACM GRID SCALE SNOW AND ICE
-      IF ( IGET(244)>0 .and. .false. ) THEN
+      IF ( IGET(248)>0 ) THEN
```

Comment:
@BenjaminBlake-NOAA The corresponding change is needed in line 975. Also could you add a new entry for ID 248 in parm/post_avblflds.xml?